docs/usage/io.rst

```diff
@@ -0,0 +1,40 @@
+Prediction Using IO Operations
+==============================
+
+This example will show you how to train a model using DFFML python API and use the
+model for prediction by taking input from stdio.
+
+The model we'll be using is ``slr`` which is a part of ``dffml`` models.
```

Comment:
It's probably sufficient to say that `slr`  comes from `dffml` and leave off the `pip install` command. Since it's in the base package, we can assume they've already installed `dffml`.

```suggestion
We'll be using the Simple Linear Regression model
(:ref:`plugin_model_dffml_slr`) which is in the ``dffml`` package.
```